package.json

```diff
@@ -92,6 +92,7 @@
     "react-addons-css-transition-group": "^15.4.1",
     "react-addons-shallow-compare": "^15.4.1",
     "react-addons-test-utils": "^15.4.1",
+    "react-burger-menu": "^2.1.4",
```

Comment:
Did you add this manually, or with `yarn add`? If you do it with yarn, it should also update the yarn.lock file.